app/controllers/hyku/invitations_controller.rb

```diff
@@ -1,33 +1,33 @@
 # frozen_string_literal: true
 
 module Hyku
-  class InvitationsController < Devise::InvitationsController
-    # For devise_invitable, specify post-invite path to be 'Manage Users' form
-    # (as the user invitation form is also on that page)
-    def after_invite_path_for(_resource)
-      hyrax.admin_users_path
-    end
+  class InvitationsController < Devise
+    class InvitationsController
```

Comment:
This formatting looks wrong here. Not sure what the rubocop error is, but you should be inheriting from this class, not creating a submodule.